client-spark/spark-2/src/main/java/org/apache/spark/shuffle/celeborn/SparkShuffleManager.java

```diff
@@ -163,8 +156,8 @@ public ShuffleBlockResolver shuffleBlockResolver() {
 
   @Override
   public void stop() {
-    if (shuffleClient != null) {
-      shuffleClient.shutdown();
+    if (isDriver()) {
```

Comment:
When will `isDriver==false` happen? And if it is true, why do we need to reset ShuffleClient?

Reply:
Reset always for unit test, follow old logic